sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -231,7 +231,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>1.10.19</version> <!-- {x-version-update;cosmos_org.mockito:mockito-core;external_dependency} -->
+      <version>3.4.6</version> <!-- {x-version-update;cosmos_org.mockito:mockito-core;external_dependency} -->
```

Comment:
Though one issue here - you updated the dependency - but didn't use it correctly here. 

There are 2 things you need to do. 

In this file - https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt
you need to remove this line - https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt#L244

And in your azure-cosmos/pom.xml - you need to use the mockito-core version mentioned here - https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt#L216
And update the tag to below . 

```
<version>3.0.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
```

If this doesn't make sense, let me know and I will explain you how this external_dependencies work in central SDK.